tests/certification/secretstores/hashicorp/vault/components/caFamily/Makefile

```diff
@@ -0,0 +1,48 @@
+
+TARGETS = certificates/key.pem certificates/cert.pem caPem/hashicorp-vault.yml certificates/incorrect-ca-key.pem certificates/incorrect-ca-cert.pem
+
+all: $(TARGETS)
+
+# Create cert and key lasting 10 years, no password,  no prompt for
+# subject details. Also set subjectAltName so we avoid the
+# "x509: certificate relies on legacy Common Name field" errors
+
+certificates/key.pem certificates/cert.pem:
```

Comment:
Rather than adding certs to the repo, can we have them auto-generated at the start of the test? I know they're all used for tests only but it's usually better to not have certs in Git to begin with.

Reply:
Sure. Good thing this is just a matter of invoking a `make` in `caFamily` directory.

This would begin a dependency on having `openssl `installed wherever you run the tests. Is that OK as a dependency?

Reply:
I don't have a problem with a dependency on openssl

Reply:
This was addressed in 97bb296ffcbda33dfc7bdd936a528acac0826cb4